tests/api/auth.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, it, expect } from 'vitest';
+import { generateRandomString } from '../../src/api/auth';
+
+describe('Auth', () => {
+	it('generateRandom String', () => {
+		const length = 16;
+		const randomString1 = generateRandomString(length);
+		const randomString2 = generateRandomString(length);
+		expect(randomString1).not.toBe(randomString2);
+		expect(randomString1.length).toBe(randomString2.length);
```

Comment:
We know that randomString1 is the same length as randomString2, but do we know that they're the expected length of..? 16? 32? 64? 42069?

Reply:
that was a typo, will update to check that the resulting length matches desired length